pytest_selenium/pytest_selenium.py

```diff
@@ -189,17 +197,10 @@ def pytest_addoption(parser):
                           '(default: %default)')
     group._addoption('--driver',
                      action='store',
+                     choices=SUPPORTED_DRIVERS,
                      dest='driver',
-                     metavar='str',
-                     help='webdriver implementation. '
-                          'Valid values are: %s.' %
-                          ', '.join(SUPPORTED_DRIVERS))
-    group._addoption('--capability',
```

Comment:
I explained that I have one more commit for this PR.. the behaviour of this will change but it will be restored. :)